packages/trader/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -88,6 +88,7 @@ export default class CFDStore extends BaseStore {
 
     // other platforms can redirect to here using account switcher's `Add` account button
     // so in that case we should open the corresponding account opening modal on load/component update
+    @action.bound
```

Comment:
and here I'm avoiding a console error 'can't read properties of undefined (reading createCFDAccount)'